packages/Emoji/index.tsx

```diff
@@ -1,22 +1,30 @@
 import React, { useMemo } from 'react'
-import { bool, number, string } from 'prop-types'
 import { Box } from '@welcome-ui/box'
 import Bowser from 'bowser'
+import { WuiProps } from 'System'
 
 const EMOJI_PATH = 'https://cdn.welcome-ui.com/emojis/'
 
 // Apple by default for SSR
 const IS_APPLE_OS =
   !process.browser || Bowser.parse(window.navigator.userAgent).platform.vendor === 'Apple'
 
-export function Emoji({
+export interface EmojiProps extends React.HTMLAttributes<HTMLButtonElement> {
```

Comment:
J'ai pas fait de TS depuis un bail mais je pense que vu qu'on retourne une image ça serait mieux avec ces types:
```suggestion
export interface EmojiProps extends React.HTMLAttributes<HTMLImageElement> {
```

Reply:
bien vu 👍🏻

Reply:
Sorry pour l'erreur d'inattention 👎